src/messages/GE.json

```diff
@@ -471,5 +471,6 @@
   "Scope consent successful": "Scope consent successful",
   "Query documentation": "Query documentation",
   "This token is not a jwt token and cannot be decoded by jwt.ms": "This token is not a jwt token and cannot be decoded by jwt.ms",
-  "The URL must contain graph.microsoft.com": "The URL must contain graph.microsoft.com"
+  "The URL must contain graph.microsoft.com": "The URL must contain graph.microsoft.com",
+  "Claims challenge": "Additional authentication requirements must be satisfied through a claims challenge"
```

Comment:
I haven't seen this message displayed or used in the code. Is the intention to leave it out?

Reply:
This was part of the initial design before changes were requested. Let me remove them also. We are not using the text anymore